protocol/entry_hash.go

```diff
@@ -0,0 +1,23 @@
+package protocol
+
+import (
+	"crypto/sha256"
+
+	"github.com/AccumulateNetwork/accumulate/smt/managed"
+)
+
+// ComputeEntryHash
+// returns the entry hash given external id's and data associated with an entry
+func ComputeEntryHash(root []byte, extIds [][]byte, data []byte) []byte {
```

Comment:
We discussed this as being simply:

`func ComputeEntryhash(data[][]byte) []byte {`

Don't we want to do this?

Reply:
yes, we do.  fixed it.